extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx

```diff
@@ -355,21 +359,22 @@ function _mapDisplaySets(
   displaySets,
   thumbnailImageSrcMap,
   trackedSeriesInstanceUIDs,
-  viewports // TODO: make array of `displaySetInstanceUIDs`?
+  viewports, // TODO: make array of `displaySetInstanceUIDs`?
+  isSingleViewport
 ) {
   const thumbnailDisplaySets = [];
   const thumbnailNoImageDisplaySets = [];
   displaySets.forEach(ds => {
     const imageSrc = thumbnailImageSrcMap[ds.displaySetInstanceUID];
     const componentType = _getComponentType(ds.Modality);
-    const firstViewportIndexWithMatchingDisplaySetUid = viewports.findIndex(
-      vp => vp.displaySetInstanceUID === ds.displaySetInstanceUID
-    );
-
-    const viewportIdentificator =
-      viewports.length > 1
-        ? _viewportLabels[firstViewportIndexWithMatchingDisplaySetUid]
-        : '';
+    const viewportIdentificator = isSingleViewport
```

Comment:
Can we stop using viewportIdentificator and use viewportIdentifier instead? Identificator is not a word